### PR TITLE
Add test case for MySQLDataTypeHandlerFactory

### LINKED ISSUE #16686: Add test case for MySQLDataTypeHandlerFactory

### PR DESCRIPTION
**Pull request by weihubeats**
Fixes #16686.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import org.apache.shardingsphere.data.pipeline.mysql.ingest.column.value.impl.MySQLUnsignedSmallintHandler;
+import org.apache.shardingsphere.data.pipeline.mysql.ingest.column.value.impl.MySQLUnsignedTinyintHandler;
+
+import org.junit.Test;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertTrue;
+
+public class MySQLDataTypeHandlerFactoryTest {
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java:32]: @weihubeats Please add final for MySQLDataTypeHandlerFactoryTest.

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Test
+    public void assertNewInstance() {
+        Optional<MySQLDataTypeHandler> result1 = MySQLDataTypeHandlerFactory.newInstance("TINYINT UNSIGNED");
+        assertTrue(result1.isPresent());
+        assertTrue(result1.get() instanceof MySQLUnsignedTinyintHandler);
+
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java:39]: @weihubeats Please remove these useless blank lines.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Comment by weihubeats** [comment]
processed

**Comment by strongduanmu** [comment]
Hi @weihubeats, can you import official idea settings file to avoid code style problem——https://shardingsphere.apache.org/community/cn/contribute/code-conduct/?

**Comment by weihubeats** [comment]
of course

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.Optional;
+
+import static org.junit.Assert.assertTrue;
+
+public final class MySQLDataTypeHandlerFactoryTest {
+
+    @Test
+    public void assertNewInstance() {
+        Optional<MySQLDataTypeHandler> result1 = MySQLDataTypeHandlerFactory.newInstance("TINYINT UNSIGNED");
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java:36]: @weihubeats Can you rename result1 to a meaningful name?

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        assertTrue(result1.isPresent());
+        assertTrue(result1.get() instanceof MySQLUnsignedTinyintHandler);
+        Optional<MySQLDataTypeHandler> result2 = MySQLDataTypeHandlerFactory.newInstance("SMALLINT UNSIGNED");
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java:39]: Same problems.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java (hunk 3 above)
Comment (weihubeats) [reply shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/MySQLDataTypeHandlerFactoryTest.java]: done

**Comment by terrymanu** [comment]
@weihubeats Could you change the PR title more make sense to describe the PR meaning?

**Comment by weihubeats** [comment]
Ok, modified to Add test case for MySQLDataTypeHandlerFactory

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**